data-raw/battles.R

```diff
@@ -18,7 +18,7 @@ battles <- tibble::tribble(~Battle, ~Date,
                            "Battle of Tora Bora","2001-12-12..2001-12-17",
                            "Battle of Shawali Kowt","2001-12-03",
                            "Battle of Sayyd Alma Kalay","2001-12-04",
-                           "Battle of Amami-Ōshima","2001-12-22",
+                           "Battle of Amami-Oshima","2001-12-22",
```

Comment:
Why is `data/battles.rda` so much smaller?

Reply:
The data-raw for battles is a small preparation script, not really a raw version of a dataset... Is there anything you would like me to do here?